tests/singlenode_with_memory_less_than_250M/singlenode_test.go

```diff
@@ -1,10 +1,7 @@
-//go:build integration
-// +build integration
-
 package test
 
 import (
-	"os"
+	"log"
```

Comment:
same comments :

- I think it should be moved into testing.M function instead and have it ready before everything
- avoid using `log`. we can use `t.LogX` instead
- null should be nil

Reply:
Please refactor the test to use planetary network only, public ips, wireguard been checked in the singlenode test